bin/sdl_make_tracklooper

```diff
@@ -57,9 +59,13 @@ if [ -z ${MAKECUTVALUES} ]; then MAKECUTVALUES=false; fi
 if [ -z ${PRIMITIVE} ]; then PRIMITIVE=false; fi
 if [ -z ${T3T3EXTENSION} ]; then T3T3EXTENSION=false; fi
 if [ -z ${DONTUSENN} ]; then DONTUSENN=false; fi
-if [ -z ${CPUBACKEND} ]; then CPUBACKEND=false; fi
+if [ -z ${ONLYGPUBACKEND} ]; then ONLYGPUBACKEND=false; fi
+if [ -z ${ONLYCPUBACKEND} ]; then ONLYCPUBACKEND=false; fi
 if [ -z ${NOPLSDUPCLEAN} ]; then NOPLSDUPCLEAN=false; fi
 
+# If using both -G and -C, -G takes priority
+if [ "${ONLYGPUBACKEND}" == true ]; then ONLYCPUBACKEND=false; fi
```

Comment:
Could we print also a message if this happens?

Also, it seems like this may be unneeded, given the `elif` order below.

Reply:
Yeah, I think it's not needed, but I agree that a message saying that that combination of flags is unintended would be good.